proto-lens/src/Data/ProtoLens/Encoding.hs

```diff
@@ -7,8 +7,11 @@ module Data.ProtoLens.Encoding (
     -- ** Delimited messages
     buildMessageDelimited,
     parseMessageDelimited,
+    decodeDelimitedMessageH,
```

Comment:
I think we should try to mirror the naming scheme of the other functions:

`decodeMessage`, `decodeMessageDelimited`, `decodeMessageDelimitedH`.

Does that sound reasonable to you?

Reply:
Ouch, silly me! I can't believe I got this wrong. Will fix.